modyn/config/schema/pipeline/config.py

```diff
@@ -30,7 +30,7 @@ class ModynPipelineConfig(ModynBaseModel):
 
     training: TrainingConfig
     data: DataConfig
-    trigger: TriggerConfig
+    trigger: TriggerConfig  # type: ignore[valid-type]
```

Comment:
why this change in mypy? it was not complaining, was it?

Reply:
It was complaining in an earlier version. I think we can remove it now though! Thanks!

Reply:
Still required, unfortunately. 
<img width="590" alt="image" src="https://github.com/eth-easl/modyn/assets/25231329/663e8f4b-3741-460e-bf3f-093efe3c0c82">